assets/pages/monthly-contributions/reducers/reducers.js

```diff
@@ -19,7 +19,10 @@ export type State = {
   amount: number,
   country: string,
   error: ?string,
+  processing: boolean,
```

Comment:
Instead of having a processing flag, would be more helpful to have a `paymentState : PaymentState` where `type PaymentState = 'Processing' | 'Failed' | 'Succeeded'`?